epistemix_platform/src/epistemix_platform/repositories/database.py

```
@@ -131,22 +132,126 @@ def create_sqlite_engine(database_url: str) -> Engine:
     )
 
 
-def create_engine_from_config(config: "Config" = None, database_url: str = None) -> Engine:
+def create_postgresql_engine_with_iam(
```

Pylint (4.0.2)

[refactor] Too many positional arguments (6/5)

(R0917)